src/main/java/org/coffeemine/app/spring/UserProfile.java

```diff
@@ -1,68 +1,81 @@
-
+        back.addClickListener(e -> UI.getCurrent().navigate(Overview.class));
+        back.addThemeVariants(ButtonVariant.MATERIAL_OUTLINED);
+        HorizontalLayout usernameTitle = new HorizontalLayout();
+        usernameTitle.getStyle().set("margin", "auto");
+        String accountName = "Account Name";
+        H2 title = new H2(accountName);
+        title.getStyle().set("padding", "0px");
+        title.getStyle().set("margin-top", "auto");
+        title.getStyle().set("margin-bottom", "auto");
+        // circle capital letter A
+        Span letter = new Span(accountName.substring(0, 1));
+        letter.addClassName("lettericon");
+        usernameTitle.add(letter, title);
+        userprofile.add(back);
+        userprofile.add(usernameTitle);
         HorizontalLayout info = new HorizontalLayout();
         info.setWidthFull();
-        this.add(info);
+        userprofile.add(info);
         // details
         VerticalLayout details = new VerticalLayout();
         details.setWidthFull();
         details.add(new H3("Account details"));
-
         FormLayout accountDetails = new FormLayout();
-        accountDetails.addFormItem(new TextField(), "Full name:");
+        accountDetails.addFormItem(new TextField(), "Full name：");
         accountDetails.addFormItem(new TextField(), "Email:");
         details.add(accountDetails);
-
         VerticalLayout states = new VerticalLayout();
         states.setWidthFull();
-        states.add(new H3("Account states"));
+        states.add(new H3("Account states "));
         FormLayout accountStates = new FormLayout();
-        accountStates.addFormItem(new Span("0.0"),"Salary per hour");
-        accountStates.addFormItem(new Span("0.0"),"Total hours worked");
+        accountStates.addFormItem(new Span("0.0"), "Hourly salary");
+        accountStates.addFormItem(new Span("0.0"), "Total hours worked: ");
         states.add(accountStates);
         info.add(details, states);
-
-        Icon clock = VaadinIcon.CLOCK.create();
-        clock.setSize("4em");
+        H3 currentProject = new H3("Current projects");
         Button save = new Button("save");
         save.getStyle().set("margin-left", "auto");
-        save.addThemeVariants(ButtonVariant.MATERIAL_CONTAINED);
-        FormLayout projectsName = new FormLayout();
-        projectsName.addFormItem(new TextField(), "Current project:");
-        add(clock);
-        add(save);
-        add(projectsName);
+        save.addThemeVariants(ButtonVariant.MATERIAL_OUTLINED);
+        HorizontalLayout currentProjects = new HorizontalLayout();
+        String projectName = "Lorem Ipsum";
+        H3 lorem = new H3(projectName);
+        lorem.getStyle().set("padding", "0px");
+        lorem.getStyle().set("margin-top", "auto");
+        lorem.getStyle().set("margin-bottom", "auto");
+        // circle capital letter L
+        LetterIcon L = new LetterIcon(projectName.substring(0, 1));
+        L.getStyle().set("font-size", "30px");
```

Comment:
Even single-letter variable names should follow the camelCase or snake_case formatting rules; that is, they should not be capitals.